environment_mac.yml

```diff
@@ -32,12 +29,16 @@ dependencies:
   - pillow>=8.0.1
   - scikit-learn
   # 3rd party algorithms for different modules
+  - cudatoolkit=10.2
   - opencv=4.5.0 # preferring conda-forge's distribution as it provides the latest distribution
   - pytorch
   # io
   - h5py
+  - plotly=4.14.3
```

Comment:
Just out of curiosity, is this the latest stable plot it release version?

Reply:
It was, when I started working on this, which is a few months ago now I think

Reply:
Cool